tests/json/arguments.rs

```diff
@@ -56,4 +59,13 @@ impl Arguments {
 
         Ok(Arguments::InsertMany { documents: docs })
     }
+
+    pub fn delete_one_from_json(object: &Object) -> Result<Arguments, String> {
+        let f = |x| Some(Bson::from_json(x));
+        let document = val_or_err!(object.get("filter").and_then(f),
+                                   Some(Bson::Document(doc)) => doc,
+                                   "`insert_one` requires document");
```

Comment:
I'll fix these ones for this pull request, although in the next one I merged the "delete_one" methods with the "delete_many", and I changed the message there just to say "delete".